common.cmake

```diff
@@ -46,3 +40,26 @@ function(nrfxlib_calculate_lib_path lib_path)
   endif()
   set(${lib_path} "lib/${arch_soc_dir}/${float_dir}${short_wchar}" PARENT_SCOPE)
 endfunction()
+
+function(get_mbedtls_dir ARM_MBEDTLS_PATH)
+  if(EXISTS ${ARM_MBEDTLS_PATH})
+  # Do nothing, just use the provided path
+  elseif(WEST)
+  ## Use `west` to find the mbedtls tree we use (this is not the
+  ## mbedtls tree distributed as a zephyr module).
+    execute_process(
+      COMMAND ${PYTHON_EXECUTABLE} -c
+      "from west.manifest import Manifest; manifest = Manifest.from_file(); print(manifest.get_projects(['mbedtls-nrf'])[0].posixpath)"
+      OUTPUT_VARIABLE west_project_output
+      RESULT_VARIABLE result
+    )
```

Comment:
> Consider using the `west list` command. It might be more readable for you:

The existing code is done on purpose to avoid calling west with a different python interpreter than the one used by the rest of the build system. Later zephyr changes make it unnecessary, but they are not yet included in NCS, so I think this should remain as-is.

Your suggested change could be made once Zephyr commit 79b1f89f7d4229d494c8b72c0acd715d61108456 is included in NCS.

Reply:
No, don't do that:
https://github.com/zephyrproject-rtos/zephyr/pull/26060

Reply:
Thank you for that update!